rocAL/rocAL/include/api/rocal_api_augmentation.h

```diff
@@ -49,7 +49,7 @@ extern "C"  RocalTensor  ROCAL_API_CALL rocalSequenceRearrange(RocalContext p_co
 /// \param resize_longer The length of the larger dimension of the image.
 /// \param interpolation_type The type of interpolation to be used for resize.
 /// \return
-extern "C"  RocalImage  ROCAL_API_CALL rocalResize(RocalContext context, RocalImage input,
+extern "C"  RocalTensor  ROCAL_API_CALL rocalResize(RocalContext context, RocalTensor input,
```

Comment:
I am assuming we don't have to go back to RocalImage after this merge.

Reply:
Yes, we will be using RocalTensor henceforth